packages/react-ui/lib/events/fixClickFocusIE.ts

```diff
@@ -1,5 +1,5 @@
-import { getClosestFocusableElement } from '../dom/getFocusableElements';
 import { isIE11, isEdge } from '../client';
+import { getClosestFocusableElement } from '../dom/tabbableHelpres';
```

Comment:
Не стоит повторять мои очепятки)
```suggestion
import { getClosestFocusableElement } from '../dom/tabbableHelpers';
```